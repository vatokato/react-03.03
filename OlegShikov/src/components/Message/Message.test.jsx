import React from 'react';
import TestRenderer from 'react-test-renderer';
import {Message} from './Message';

describe("Message.jsx", () => {
    it("render content", () => {
        const props = {
            name: "Ivan",
            content: "Hello!",
        };

        const element = TestRenderer.create(<Message  {...props}/>)

        expect(element.toJSON()).toMatchSnapshot();
    });
})
