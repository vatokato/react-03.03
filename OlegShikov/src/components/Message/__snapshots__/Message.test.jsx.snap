// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message.jsx render content 1`] = `
<li
  className="Message"
>
  <strong>
    Ivan
    :
  </strong>
   
  Hello!
</li>
`;
